great_expectations/datasource/fluent/file_path_data_asset.py

```diff
@@ -182,6 +182,11 @@ def build_batch_request(
         Returns:
             A BatchRequest object that can be used to obtain a batch list from a Datasource by calling the
             get_batch_list_from_batch_request method.
+
+        Note:
+            Option "batch_slice" is supported for all DataAsset extensions of this class "_FilePathDataAsset(DataAsset)"
```

Comment:
I would remove the reference to `_FilePathDataAsset` since it is private. This method should be inherited by all subclasses and appear in the API docs there. I would just say batch_slice is supported by this class.

Reply:
I took care of it -- thank you.